utils/superset/db_name_change.py

```diff
@@ -0,0 +1,52 @@
+from io import IOBase
+import os
+import json
+import argparse
+import logging
+from pyexpat import ErrorString
+import sys
+
+
+DATABASE_NAME = "template_database_name"
+
+# create logger
+logging.basicConfig(level=logging.DEBUG)
+
+
+# file extension validation
+def validate_extension(file_name):
+    _, ext = os.path.splitext(file_name)
+    if ext.lower() != ('.json') or len(ext) == 0:
+        raise argparse.ArgumentTypeError('File must have JSON extension')
+    return True
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-d", "--database_name", required=True, type=str)
```

Comment:
I would change this to parser.add_argument("database_name")

Reply:
Done.